template/{% if sphinx %}docs{% endif %}/conf.py.jinja

```diff
@@ -108,7 +108,7 @@ inheritance_graph_attrs = {"rankdir": "TB"}
 # Common links that should be available on every page
 rst_epilog = """
```

Comment:
Since #51 this `rst_epilog` is no longer needed, so the entire variable can be deleted from this file